R/taxonomy.R

```diff
@@ -135,7 +135,6 @@ setGeneric("taxonomyRanks", signature = c("x"),
              standardGeneric("taxonomyRanks"))
 
 #' @rdname taxonomy-methods
-#' @aliases taxonomicRanks
 #'
```

Comment:
It said that `taxonomyRanks` had an alias called `taxonomicRanks`, but it wasn't working. Should we have an alias?

Reply:
In this case I think it would be more clear to have one unique name. Should be checked to OMA as well.

Reply:
I checked OMA, there is used only `taxonomyRanks`, not `taxonomicRanks` so that's correct